_posts/2015-09-15-rnd-day-2015.md

```diff
@@ -0,0 +1,115 @@
+
+Team Red presented the architecture of the App & Theme Store.
+The Store allows merchants to customise the design of their shop with certain themes.
+Furthermore, they can add additional functionalities to their shop by installing apps developed by external developers using the ePages APIs.
+
+{% image blog-rnd-appstore.jpg %}{% endimage %}
+
+They explained the different app types, the store currently supports:
+
+* REST apps that use the ePages [REST API](https://developer.epages.com/)
+* SOAP apps that use the ePages [SOAP API](https://developer.epages.com/soap/index.html)
+* Themes and Affiliates
+
+The challenge was to provide an architecture which not only allows external developers to develop and submit their app easily, but also to provide an administration tool for the product management which allows them to distribute the apps to the different providers.
+
+Despite the brief time frame of the session and the complexity of the architecture, Team Red succeeded
+to present in an understandable way using several diagrams and live demos of the different App & Theme Store use cases:
+
+- developers point of view: How an app developer can submit an app.
+- Product Management's point of view: How PM is able to review an app and manage apps for the different shop types.
+- merchants point of view: How to install and uninstall an app in the merchant's administration area.
+
+# Spotlight: New Storefront
+
+{% image blog-rnd-unity.jpg %}{% endimage %}
+
+Team Unity shared insights on the new storefront and editor.
+They explained how they plan to integrate this into the existing ePages product using the ePages [REST API](https://developer.epages.com/).
+
+Using [node.js](https://nodejs.org/en/) and [akka](http://akka.io/), [React](http://facebook.github.io/react/) and [webpack](https://webpack.github.io/) they realised a cool and slick implementation.
+
+# Hands-on: Microservices
+
+In this workshop, the participants gained first experience on the new ePages architecture.
+They completed hands-on exercises illuminating common microservice patterns, like synchronous and asynchronous communication and persistence - and had fun building a microservice-based [pizza bakery](https://github.com/ePages-de/rnd-microservices-handson).
+
+# Spotlight: Automating Jenkins
+
+Check out our blog post [Infrastructure as code: automating Jenkins](https://developer.epages.com/blog/2015/06/25/infrastructure-as-code.html) to find out more on this topic.
+Stay tuned for an upcoming update on this topic.
+
+# Spotlight: Test Automation
+
+Ideally, new features and bug fixes are available on the live systems as soon as they are implemented and verified.
+In order to achieve that, there is a clear necessity for [automated tests](http://blog.daverooney.ca/2015/04/getting-started-with-test-driven.html).
+For each application layer, specific test automation tools and techniques exist:
+
+* The Presentation layer is what the user gets to see.
```

Comment:
Remove the full stop
You also have used lower case for the first letter of the bullet point with the others?